simulator/mips/mips_instr.cpp

```diff
@@ -231,44 +231,65 @@ void FuncInstr::initFormat()
 
     if ( is_R)
     {    
-        const auto& entry = isaMapR.at( ident);
-        if ( entry != isaMapR.end())
+        try
+        { 
+            const auto& entry = isaMapR.at( ident);
```

Comment:
We do not use C++ exceptions according to our code guidelines.
There is a nice function `std::unordered_map::find`. It returns an iterator to element if it exists an iterator to `end()` if it does not.

Reply:
I understand